app/lib/navigation/state_file_az_question_navigation.rb

```diff
@@ -44,11 +44,11 @@ class StateFileAzQuestionNavigation < Navigation::StateFileBaseQuestionNavigatio
         Navigation::NavigationStep.new(StateFile::Questions::TaxesOwedController),
         Navigation::NavigationStep.new(StateFile::Questions::TaxRefundController),
         Navigation::NavigationStep.new(StateFile::Questions::EsignDeclarationController), # creates EfileSubmission and transitions to preparing
-      ]),
+      ], false, true),
```

Comment:
just highlighting something that i mentioned in the story that `StateFile::Questions::PostDataTransferController` can be currently in a state where df data doesnt exist -- it redirects the user to the data-import failed page if it failed